guides/source/active_record_validations.md

```diff
@@ -1253,6 +1265,73 @@ end
 The validation only runs when all the `:if` conditions and none of the `:unless`
 conditions are evaluated to `true`.
 
+Strict Validations
+------------------
+
+You can also specify validations to be strict and raise
+`ActiveModel::StrictValidationFailed` when the object is invalid.
```

Comment:
Do you think it's worth adding a sentence here giving an example of why you might want a validation to be strict vs not?

Reply:
Added a paragraph to describe this.